sdk/core/switcher/trusted-stack-assembly.h

```diff
@@ -19,28 +19,39 @@ EXPORT_ASSEMBLY_OFFSET(TrustedStack, c12, 12 * 8)
 EXPORT_ASSEMBLY_OFFSET(TrustedStack, c13, 13 * 8)
 EXPORT_ASSEMBLY_OFFSET(TrustedStack, c14, 14 * 8)
 EXPORT_ASSEMBLY_OFFSET(TrustedStack, c15, 15 * 8)
-EXPORT_ASSEMBLY_OFFSET(TrustedStack, mstatus, 16 * 8)
-EXPORT_ASSEMBLY_OFFSET(TrustedStack, mcause, (16 * 8) + 4)
-#ifdef CONFIG_MSHWM
-EXPORT_ASSEMBLY_OFFSET(TrustedStack, mshwm, 17 * 8)
-EXPORT_ASSEMBLY_OFFSET(TrustedStack, mshwmb, (17 * 8) + 4)
+#ifdef CHERIOT_HAS_ZTOP
+EXPORT_ASSEMBLY_OFFSET(TrustedStack, ztop, 16 * 8)
+#	define TSTACK_HAS_ZTOP 1
```

Comment:
This is _extremely_ minor, but the differing "types" of `CHERIOT_HAS_ZTOP` and `TSTACK_HAS_ZTOP` is bugging me.  Maybe `TSTACK_ZTOP_WORDS` or such?